docs/conf.py

```diff
@@ -0,0 +1,30 @@
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+"""Configuration file for the Sphinx documentation builder."""
+
+
+# -- Project information -----------------------------------------------------
+
+project = "pip-tools"
+author = "Jazzband Contributors"
+copyright = f"2021, {author}"
```

Comment:
No Jazzband specific policy, but the first year of creation is sensible to indicate when the project was started. It shouldn't be "Jazzband Contributors" but "pip-tools contributors" as well.

Reply:
Ah, okay. According to my prior research, the copyright year of the content creation should be stated and then, the years of the updates to the content (list or range). Using the current year (and updating it every January) is incorrect because this reflects the invalid info.

OTOH my understanding is that the copyright notice by itself doesn't have any legal power — it's mostly informational and the copyright is assigned to the (parts) of the content regardless of what that string says.

But because many people interpret it differently, such a statement tends to be incorrect and/or out-of-date. This is why the recent recommendations that I saw were not to use it at all (for the ease of maintenance). *I think* somebody from RH legal noticed this approach and also the Linux Foundation recommends it: https://www.linuxfoundation.org/blog/copyright-notices-in-open-source-software-projects/.

P.S. Of course, IANAL so takes this with a grain of salt. Still, I do like the recommendation not to state the year in the copyright lines.


-----------------


As for "community contributors" vs "project contributors", I saw both. But I don't have a problem with just accepting what you prefer. Just wanted to put this comment in the public, for future reference.

Reply:
> Ah, okay. According to my prior research, the copyright year of the content creation should be stated and then, the years of the updates to the content (list or range). Using the current year (and updating it every January) is incorrect because this reflects the invalid info.
> 
> OTOH my understanding is that the copyright notice by itself doesn't have any legal power — it's mostly informational and the copyright is assigned to the (parts) of the content regardless of what that string says.
> 
> But because many people interpret it differently, such a statement tends to be incorrect and/or out-of-date. This is why the recent recommendations that I saw were not to use it at all (for the ease of maintenance). _I think_ somebody from RH legal noticed this approach and also the Linux Foundation recommends it: https://www.linuxfoundation.org/blog/copyright-notices-in-open-source-software-projects/.
> 
> P.S. Of course, IANAL so takes this with a grain of salt. Still, I do like the recommendation not to state the year in the copyright lines.

Sure that works for me, let's skip the year number, the changelog provides ample reference to the updates anyway.

> As for "community contributors" vs "project contributors", I saw both. But I don't have a problem with just accepting what you prefer. Just wanted to put this comment in the public, for future reference.

I'd prefer to point out that not all community members are automatically contributors to the project. And pip-tools has had contributors before it became a community project. So it's easiest to refer to the contributors as "pip-tools contributors" IMO.

Reply:
Fair enough. This will be addressed via https://github.com/jazzband/pip-tools/pull/1467.